kafka-connect-cassandra/src/test/scala/com/datamountaineer/streamreactor/connect/cassandra/source/TestCqlGenerator.scala

```diff
@@ -87,15 +87,16 @@ class TestCqlGenerator extends WordSpec with Matchers with BeforeAndAfter with M
   }
 
   def configureMe(kcqlIncrementMode: String, configIncrementMode: String): CassandraSourceSetting = {
-    val myKcql = s"INSERT INTO kafka-topic SELECT string_field, timestamp_field FROM cassandra-table PK timestamp_field $kcqlIncrementMode"
+    val myKcql = s"INSERT INTO kafka-topic SELECT string_field, timestamp_field FROM cassandra-table PK timestamp_field BATCH=200 $kcqlIncrementMode"
     val configMap = {
       Map(
         CassandraConfigConstants.KEY_SPACE -> CASSANDRA_KEYSPACE,
         CassandraConfigConstants.SOURCE_KCQL_QUERY -> myKcql,
         CassandraConfigConstants.ASSIGNED_TABLES -> s"$TABLE3",
         CassandraConfigConstants.IMPORT_MODE -> CassandraConfigConstants.INCREMENTAL,
         CassandraConfigConstants.POLL_INTERVAL -> "1000",
-        CassandraConfigConstants.FETCH_SIZE -> "200",
+        CassandraConfigConstants.FETCH_SIZE -> "500",
+        CassandraConfigConstants.BATCH_SIZE -> "800",
         CassandraConfigConstants.TIMESTAMP_TYPE -> configIncrementMode).asJava
```

Comment:
set the values for FETCH & BATCH size in the test here w/ numbers different than KCQL to test/confirm that the KCQL value is being used